catalogs/catalog-lakehouse-iceberg/build.gradle.kts

```diff
@@ -138,16 +138,10 @@ tasks {
 }
 
 tasks.test {
-  val skipUTs = project.hasProperty("skipTests")
```

Comment:
why remove this?

Reply:
It didn't work in this way (because miss `useJUnitPlatform()`) and I fixed this in root kts:
https://github.com/apache/gravitino/pull/4827/files#diff-c0dfa6bc7a8685217f70a860145fbdf416d449eaff052fa28352c5cec1a98c06R207-R213

Reply:
ok